app/models/photo.rb

```diff
@@ -0,0 +1,12 @@
+class Photo < ActiveRecord::Base
+  has_attached_file :image, default_url: "/images/:style/missing.png", styles: { original: { height: "100%", width: "100%" } , medium:"300x300>", thumb:"100x100>" }
```

Comment:
Metrics/LineLength: Line is too long. [164/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)